telemetry/src/ws.rs

```diff
@@ -2,7 +2,7 @@
 #![allow(clippy::std_instead_of_core, clippy::std_instead_of_alloc)]
 use std::time::Duration;
 
-use chrono::Local;
+use chrono::Utc;
```

Comment:
Can be reverted in next PR because what I fixed by this change was [due to chrono 0.4.20](https://docs.rs/chrono/0.4.20/src/chrono/offset/local/unix.rs.html#52) and [fixed in 0.4.22](https://docs.rs/chrono/0.4.22/src/chrono/offset/local/unix.rs.html#58).
We seem to have not noticed because this was behind another failure (ui_fail/no_conversion.stderr)